docs/reference/cli.md

```diff
@@ -9,16 +9,18 @@ redirect_from:
   - ./commands.html
 ---
 
-# Commands (CLI) Reference
+# `lakectl` CLI Reference
 {:.no_toc}
 
-{% include toc.html %}
+`lakectl` is a command-line tool to explore manage and work with lakeFS.
 
-## Installing the lakectl command locally
+{% include toc.html h_max=2 %}
 
-`lakectl` is distributed as a single binary, with no external dependencies - and is available for MacOS, Windows and Linux.
+## Installing `lakectl` locally
```

Comment:
```suggestion
## Installing lakectl locally
```
No need to codify "lakectl". It's the name of the tool, not only the name of the command (inspiration: [kubectl](https://kubernetes.io/docs/reference/kubectl/)).